src/modules/auth/components/SignUp/SignUpSection.component.tsx

```diff
@@ -1,19 +1,24 @@
+import { useTranslation } from 'react-i18next';
 import { FormSignUp } from './FormSignUp.component';
+import { Link } from 'react-router-dom';
 
 export function SignUpSection(): JSX.Element {
+  const { t } = useTranslation();
   return (
     <div className="md:flex md:justify-center">
       <div className="flex flex-col justify-between py-6 lg:w-1/2">
+        <h2 className="pb-4 text-4xl">{t('auth:signUp')}</h2>
         <FormSignUp />
 
-        <label className="label">
+        <div className="label">
           <span className="label-text">
-            You already have an account?{' '}
-            <a href="#" className="cursor-pointer text-primary">
-              Log In
-            </a>
+            {t('auth:haveAccString')}{' '}
+            <Link to="/auth/login" className="cursor-pointer text-primary">
+              {t('auth:haveAccLink')}
```

Comment:
This could be  `auth:singup.loginLink` or `auth:singup.login`, since the link is actually `/auth/login`...